packages/lib/src/components/internal/SecuredFields/lib/CSF/extensions/createSecuredFields.ts

```diff
@@ -198,6 +198,15 @@ export function setupSecuredField(pItem: HTMLElement): Promise<any> {
 
                 if (window._b$dl) console.log('### createSecuredFields::onIframeLoaded:: this.state.iframeCount=', this.state.iframeCount);
 
+                // One of our existing securedFields has just loaded new content!
+                if (this.state.iframeCount > this.state.numIframes) {
+                    this.destroySecuredFields();
+                    throw new Error(
+                        `ERROR: One or more securedFields has just loaded new content. This should never happen. securedFields have been removed.
+                        iframe load count=${this.state.iframeCount}. Expected count:${this.state.numIframes}`
+                    );
+                }
```

Comment:
Maybe you could use `AdyenCheckoutError` model here instead of the generic one?

Reply:
But hat's more for formatting an error to be passed to the merchant defined `onError` handler - what we want here is a proper big, breaking, console error

Reply:
But I guess you can throw them too... so, ok